install/kubernetes/helm/istio-remote/Chart.yaml

```diff
@@ -1,6 +1,7 @@
 apiVersion: v1
 name: istio-remote
 version: 0.8.0
```

Comment:
@sdake @costinm do you have any comments for how shall we define the initial `version` for the Chart?

Reply:
I personally feel that the version of the chart should match the version of Istio.  Revs to the helm chart version should be done as follows:

`version: 0.8.0-1`
`version: 0.8.0-2`

Reply:
@sdake I think we still need follow up PR to update `version` to `0.8.0-1` for all charts in istio?